content/knowledge/roadmap/timeline.en.yaml

```diff
@@ -254,7 +254,7 @@
     longer wants to be in association with the Foundation
 -
   key: classicRebuilds
-  link: https://ethereumclassic.netlify.com/blog/2016-08-18-meetup
+  link: https://ethereumclassic.org/blog/2016-08-18-meetup
```

Comment:
For links that rare on this site, please use relative links, in this case

```
link: /blog/2016-08-18-meetup
```

Reply:
This is for a couple of reasons:

1. The site will automatically update these links for translated links `/` -> `/de/` when i18n is on
2. External links are opened in a new tab, but it's nicer just to navigate on the same tab

Reply:
resolved